### PR TITLE
WIP Init script

### PR DESCRIPTION
**Pull request by DavidBadura**
todos from feedback

* [x] optional overwrite of existing .simpspector.yml
* [x] optional overwrite when gadget config file already exists
* [x] default enable question to no, if binary missing
* [x] enable feedback (same line, new line?)
* [x] refactor ConfigurationFile-Struct to gadget-namespace 

simpspector-analyser-init

### CODE REVIEW

**Comment by tolry** [comment]
Does anybody have sample config files for one of our Gadgets, they would like to see to be used? @simpspector/core

**Comment by tolry** [comment]
@simpspector/core I think this is in a state to be merged - opinions?

I would suggest doing additional DefaultConfigFiles after merging this. Currently we have one Gadget supporting this as a proof of concept that the process works.

File: src/Struct/GadgetConfigurationFile.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * @author Tobias Olry <tobias.olry@gmail.com>
+ */
+
+namespace SimpSpector\Analyser\Struct;
```
Comment (DavidBadura) [inline src/Struct/GadgetConfigurationFile.php]: I don't like generic namespaces like struct, handler, services and adapter with none context. I think it's better to move this class into the Gadget namespace.

Comment (tolry) [inline src/Struct/GadgetConfigurationFile.php]: @DavidBadura you're right, that fits better, will refactor this week

Comment (tolry) [inline src/Struct/GadgetConfigurationFile.php]: done

**Comment by tolry** [comment]
@simpspector/core anything more? Ready to merge?

**Comment by DavidBadura** [comment]
:+1: LGTM

**Pull request merged**